src/Host/Broker/Impl/Sessions/SessionState.cs

```diff
@@ -1,9 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using System;
-
-namespace Microsoft.R.Host.Broker.Pipes {
-    public interface IOwnedMessagePipeEnd : IMessagePipeEnd, IDisposable {
+namespace Microsoft.R.Host.Broker.Sessions {
+    public enum SessionState {
+        Running,
+        Dormant,
```

Comment:
Never used currently. Will be used for sessions that have saved state in the future.